prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir

```diff
@@ -2284,32 +2257,23 @@
         "text" : "BLOOD CULTURE GRAM STAIN"
       },
       "subject" : {
-        "reference" : "Patient/7806900e-8e0c-4e1d-bd7a-648fa966d935"
+        "reference" : "Patient/f39a0f20-ba1f-48f2-934f-35e2a3c51f10"
       },
       "encounter" : {
-        "reference" : "Encounter/011de763-d0e8-49fc-a662-e5cc348d5660"
+        "reference" : "Encounter/ebcf14ca-78e2-486c-860a-47df285253a2"
       },
       "effectiveDateTime" : "2028-08-02T02:52:01-06:00",
       "issued" : "2021-08-10T06:25:00-06:00",
       "performer" : [ {
-        "reference" : "PractitionerRole/2e6242e6-d26c-4fe3-a218-ea1b8b00e3a4"
-      } ],
-      "valueString" : "YEAST",
-      "interpretation" : [ {
-        "coding" : [ {
-          "system" : "http://terminology.hl7.org/CodeSystem/v2-0078",
-          "code" : "A",
-          "display" : "Abnormal"
```

Comment:
This got removed.  Same field as the previous one I think.

Reply:
This one's back too!